src/components/Collection/Collection.tsx

```diff
@@ -0,0 +1,245 @@
+import React, { useState } from "react";
+import {
+  Box,
+  Breadcrumbs,
+  Flex,
+  Footer,
+  Form,
+  Heading,
+  Hero,
+  HorizontalRule,
+  Pagination,
+  SimpleGrid,
+  Template,
+  TemplateBreakout,
+  TemplateContent,
+  TemplateContentPrimary,
+  TemplateContentTop,
+  TemplateFooter,
+} from "@nypl/design-system-react-components";
+import { collectionSortMap } from "~/src/constants/sorts";
+import {
+  toApiCollectionQuery,
+  toLocationQuery,
+} from "~/src/util/apiConversion";
+import { defaultBreadcrumbs } from "~/src/constants/labels";
+import { truncateStringOnWhitespace } from "~/src/util/Util";
+import { MAX_TITLE_LENGTH } from "~/src/constants/editioncard";
+import {
+  CollectionQuery,
+  CollectionQueryDefaults,
+  CollectionResult,
+} from "~/src/types/CollectionQuery";
+import CollectionUtils from "~/src/util/CollectionUtils";
+import ResultsSorts from "../ResultsSorts/ResultsSorts";
+import { useRouter } from "next/router";
+import { CollectionItemCard } from "../CollectionItemCard/CollectionItemCard";
+import Link from "../Link/Link";
+
+const Collection: React.FC<{
+  collectionQuery: CollectionQuery;
+  collectionResult: CollectionResult;
+}> = (props) => {
```

Comment:
Nit: we can destructure props here, i.e. `({ collectionResult, collectionQuery })` and then further destructure `collectionResult` within the component.